test/functional/validation-functions-and-decorators.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Please do not disable rules globally. Use `eslint-disable-next-line`.

Reply:
Because it's a test, I would say it even can be removed and the names of keys can be fixed.

Reply:
We should use camelcase everywhere in our code. This break rule is only because of validatorjs interfaces so we should disable only lines where validatorjs snake case style is used.

Reply:
Resolved.